tessera.lic

```diff
@@ -0,0 +1,62 @@
+=begin
+  Documentation:  Tessera script based upon concept around Almanac.lic. 
+                  Tries to ask the tessera about invest every 375 seconds (6 mins 15 seconds).
+=end
+
+custom_require.call(%w[common common-items])
+
+class Tessera
+  include DRC
+  include DRCI
```

Comment:
One thing we're trying to do with scripts moving forward is avoid the include lines in favor of the explicit prefix before methods.The primary reason is that it makes the code much more approachable for new contributors. It makes it easy to see where our helper methods live, and easy to go and reference them.

For, for example, you'd remove the `include DRC` line in favor of calling bput by using `DRC.bput` -- and any other methods that live there. Note: the `custom_require` line is still required.